spec/services/analytics/analytics_service_spec.rb

```diff
@@ -2,9 +2,12 @@
 
 RSpec.describe AnalyticsService, type: :service do
```

Comment:
It's a bit misleading that the test is in the `analytics` directory, while the service itself is just in the `services`.

Reply:
Good catch, I'm going to move it